### ISSUE TITLE
<hr> 's seem to go invisible at smaller breakpoints.

### ISSUE THREAD

**Issue by KingScooty** [jakegiltsoff/sassline#47]
`background-size: 100% 0.15rem;`

They seem to get too short to be visible at smaller breakpoints.

**Comment by jakegiltsoff** [comment]
@KingScooty Ah hmm, this will depend on the base values you have. I should probably just make this 1px for sanity.

**Comment by KingScooty** [comment]
@jakegiltsoff Yeah, this is using the vanilla base values fresh from a bower install, so it's default behaviour.

**Comment by jakegiltsoff** [comment]
@KingScooty Ok, well then yea it depends on the browser and screen as to wether it will be rendered. I’ll update it to 1px.
